app/services/nomis/custody/api.rb

```diff
@@ -1,5 +1,8 @@
 module Nomis
   module Custody
+    ApiObjectResponse = Struct.new(:data)
+    ApiListResponse = Struct.new(:meta, :data)
```

Comment:
I wonder if what are returning here is actually a ApiResponse and an _ApiPaginatedResponse_?  mmm ... I guess it may not always be paginated though ...

Reply:
I think the presence of the meta part for list calls means it sort of is paginated, even if there are no pages - will update the names.